django_ses/management/commands/blacklist.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# encoding: utf-8
+from optparse import make_option
+
+from django.core.management.base import BaseCommand
+
+from django_ses import models
+
+
+def _add_options(target):
+    return (
+        target(
+            '-a',
+            '--add',
+            dest='add',
+            default=False,
+            help='Adds an email to your blacklist'
+        ),
+        target(
+            '-d',
+            '--delete',
+            dest='delete',
+            default=False,
+            help='Removes an email from your blacklist'
+        ),
+        target(
+            '-l',
+            '--list',
+            dest='list',
+            default=False,
+            action='store_true',
+            help='Outputs all blacklisted emails'
+        ),
+        target(
+            '-s',
+            '--search',
+            dest='search',
+            default=False,
+            help='Search for blacklisted emails'
+        ),
+    )
+
+
+class Command(BaseCommand):
+    """Add, delete or list blacklisted email addresses"""
+
+    if hasattr(BaseCommand, 'option_list'):
+        # Django < 1.10
+        option_list = BaseCommand.option_list + _add_options(make_option)
+    else:
+        # Django >= 1.10
+        def add_arguments(self, parser):
+            _add_options(parser.add_argument)
+
+    def handle(self, *args, **options):
+        verbosity = options.get('verbosity', 0)
+        email_to_add = options.get('add', '')
+        email_to_delete = options.get('delete', '')
+        list_emails = options.get('list', False)
+        search_email = options.get('search', '')
```

Comment:
Curious why not have these as arguments?

```suggestion
    def handle(self, *args, verbosity=0, email_to_add='', etc):
```

Reply:
Same reason as previous comment 😅 I just blindly copied the code from the other command, didn't want to introduce too many changes. Let me change it!